README.md

```diff
@@ -13,9 +13,11 @@ Releasing new images is no more bound to WildFly server releases. Releases are d
 
 An image containing all you need to run a WildFly Server. This image is to be used in a docker build to install a WildFly server.
 
-* JDK11 based [runtime image](wildfly-runtime-image/image.yaml): `docker pull quay.io/wildfly/wildfly-runtime-jdk11:latest`
+* JDK11 based [runtime image](wildfly-runtime-image/image.yaml): `docker pull quay.io/wildfly/wildfly-runtime:latest-jdk11`
 
-* JDK17 based [runtime image](wildfly-runtime-image/jdk17-overrides.yaml): `docker pull quay.io/wildfly/wildfly-runtime-jdk17:latest`
+* JDK17 based [runtime image](wildfly-runtime-image/jdk17-overrides.yaml): `docker pull quay.io/wildfly/wildfly-runtime:latest-jdk17`
+
+* Latest JDK based runtime image: `docker pull quay.io/wildfly/wildfly-runtime:latest`
```

Comment:
maybe mention here that `latest` points to the more recent LTS JDK.

In other words, if we do a jdk19, `latest` will still point to `jdk17`.